### ISSUE TITLE
ghcjs-boot --dev fails on Windows.

### ISSUE THREAD

**Issue by BethAr** [ghcjs/ghcjs#339]
I'm running from Mingw32 (Msys32) Here is the error message:

```
C:\Users\Beta\AppData\Roaming\ghcjs\i386-mingw32-0.1.0-7.10.1\ghcjs\ghcjs-boot\utils\genprimopcode$ [ghc]: C:\Haskell\MinGHC-7.10.1\ghc-7.10.1\bin\ghc.exe "-o" "genprimopcode" "-O" "Main.hs" "+RTS" "-K128M"
C:\Haskell\MinGHC-7.10.1\ghc-7.10.1\bin\ghc.exe -o genprimopcode -O Main.hs +RTS -K128M
[1 of 5] Compiling ParserM          ( ParserM.hs, ParserM.o )

ParserM.hs:26:10:
    No instance for (Applicative ParserM)
      arising from the superclasses of an instance declaration
    In the instance declaration for `Monad ParserM'
ghcjs-boot.exe: error running: C:\Haskell\MinGHC-7.10.1\ghc-7.10.1\bin\ghc.exe -o genprimopcode -O Main.hs +RTS -K128M
exit status: 1
stderr:
ParserM.hs:26:10:
    No instance for (Applicative ParserM)
      arising from the superclasses of an instance declaration
    In the instance declaration for `Monad ParserM'
```

**Comment by asciiascetic** [comment]
Check out this (resolved hurrah!) issue: https://github.com/ghcjs/ghcjs/issues/335

In the README under the **Build The Libraries** section, it is suggested:

> if you are doing a development build with GHC 7.10, you need to tell `ghcjs-boot` to use the `ghc-7.10` branch of the `ghcjs-boot` repository:
> $ ghcjs-boot --dev --ghcjs-boot-dev-branch ghc-7.10

Might try adding that `--ghcjs-boot-dev-branch ghc-7.10` switch?

**Comment by BethAr** [comment]
Thank you. I completely misunderstood that line.

**Comment by manny-fp** [comment]
My issue was totally separate, it turns out that the `transformers` commit referenced by https://github.com/ghcjs/ghcjs-boot does not build with GHC 7.8.4, although the released version of transformers-0.4.3.0 does (commit ad784bdb801116d396d6f0f2020d4ece5b26f79e is the last to build, and a10a5b0bba4e720b1d29b621856ad851565c9146 breaks it).
